### ISSUE TITLE
WP Build: Page crashes when Gutenberg is not activated

### ISSUE THREAD

**Issue by dhasilva** [WordPress/gutenberg#75704]
### Description

When a plugin uses `@wordpress/build` on a site where Gutenberg is not installed, the pages fail to load with a console error: `Uncaught (in promise) TypeError: The specifier “@wordpress/a11y” was a bare specifier, but was not remapped to anything. Relative module specifiers must start with “./”, “../” or “/”.`

### Step-by-step reproduction instructions

1. Install a plugin that uses wp-build and registers a page (https://github.com/dhasilva/minimal-wp-build-test is a minimal plugin made to test this).
2. Go to its page (Tools → Minimal WP-Build Test with the test plugin above)
3. Check that it loads correctly if Gutenberg is installed, but fails to load if Gutenberg is not activated

### Screenshots, screen recording, code snippet

<img width="1244" height="367" alt="Image" src="https://github.com/user-attachments/assets/74fda356-47c5-47dd-b789-16043fe79075" />

### Environment info

"@wordpress/build": "0.7.0"
No Gutenberg installed

### Please confirm that you have searched existing issues in the repo.

- [x] Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

- [x] Yes

### Please confirm which theme type you used for testing.

- [x] Block
- [ ] Classic
- [ ] Hybrid (e.g. classic with theme.json)
- [ ] Not sure
